src/profile/Login.js

```diff
@@ -37,12 +37,7 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const Login = ({
-  initialFetch,
-  setCloseLoading,
```

Comment:
I like this - we really don't need the `setCloseLoading`, much better to just have `setLoading` take a bool parameter.